fofix/core/Theme.py

```diff
@@ -2004,9 +2004,9 @@ def renderAlbumArt(self, scene):
                 i = scene.selectedIndex
                 label = scene.itemLabels[i]
                 if label == "Random":
-                    label = scene.img_random_label
+                    label = scene.img_random_label.texture
                 if not label:
-                    label = scene.img_empty_label
+                    label = scene.img_empty_label.texture
```

Comment:
Idem, you should check if the variable is not None.

Reply:
Fixed.